data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2721 @@
+			choice
+				`	"Could you help me translate some other times?"`
+				`	(Let her leave.)`
+					goto leave
+			`	"I am not even sure I will stay in Unfettered space, but if we meet again, maybe. I do not want to lose any status I might acquire by speaking with you. Some among us are not fan of humans."`
+			label leave
+			`	The crowd, including the young girl, leave, as do the Chiefs later on. You notice one of them looking at you, pensive, as he has just noticed your presence.`
+				goto next
+			label continue
+			`	It is now the turn of a child to come, and he speaks very quietly, as if afraid. He appears lost. One of the Chief comes close to him and reassures him before speaking to his request. <thirdgroup> Progressively, every single Hai stands up, and say what you're pretty sure means "I do" in Hai. The Chief seems very moved by this, as a parent would be to see his child succeed in life. He then designates someone, and the child goes to them.`
+			`	The sun is almost set, and you can see that the Chiefs are growing a bit impatient. One Hai then arrives; he is probably the last one to pass today. <fourthgroup> Then they rise up, and one speaks to the crowd. <closing> At this, the crowd, including the young girl, leave, as do the Chiefs later on. You notice one of them looking at you, pensive, as he has just noticed your presence.`
+			label next
+			`	The Chief moves toward you. He seems to be lost in his thoughts and is mumbling something about you. In your own language, he says, "We do not get alien visitors often here. There has to be a reason for your presence..." When he speaks again, directly to you, it is with more focus. "Sorry, I was just thinking out loud! Well, you are here anyway, so maybe you could help us.`
+			`	"We have friends outside of our space, civilians and others. You see, we would do anything to help a friend, and a few Hai want to join us. I can only hope you are willing to help us reunite with our brothers and sisters from Hai space," he explains, apparently a bit desperate, considering he knows very little, if anything at all, about you.`
+			choice
+				`	"Of course. What do you need me to do?"`
+					goto action
+				`	"What do you mean, brothers and sisters from Hai space? Are you not different?"`
+					goto brethren
+				`	"Why do you need me to do it?"`
+			label whyme
+			`	"Well, normally they would be able to pay to come here, but the price is quite high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed they have a fleet on the border between us. And not many Hai captains are willing to come here. If they do, they charge large fees because of the obvious risks involved - for instance, getting killed in the cross fire. Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
```

Comment:
```suggestion
			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not many Hai captains are willing to come here. If they do, they charge large fees because of the obvious risks involved - for instance, getting killed in the cross fire. Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
```